### ISSUE TITLE
no logs

### ISSUE THREAD

**Issue by abelal83** [vmware-tanzu/octant#729]
**What steps did you take and what happened:**
Clicked on logs tab to view logs however, I only see "No logs"
When viewing the same pod with kubectl logs -f I can see logs
I've tried ticking and unticking "Display timestamp" with no luck

**What did you expect to happen:**
To view logs

**Anything else you would like to add:**
logs from octant
2020-02-25T12:02:10.308Z        INFO    api/api.go:99   unable to serve {"code": 500, "message": "check if container has started: get pod serene-web-5b76858fcf-cb8rd in serene-dev: pods \"serene-web-5b76858fcf-cb8rd\" not found"}
W0225 12:02:11.088859   10764 reflector.go:302] pkg/mod/k8s.io/client-go@v0.0.0-20190620085101-78d2af792bab/tools/cache/reflector.go:98: watch of *unstructured.Unstructured ended with: too old resource version: 100771155 (100773780)

states the pod can't be found which is not valid.

S C:\Users\abx> kb get pod serene-web-5b76858fcf-cb8rd -n serene-dev
NAME                          READY   STATUS    RESTARTS   AGE
serene-web-5b76858fcf-cb8rd   1/1     Running   0          4d23h

**Environment:**

- Octant version (use `octant version`): 0.10.2
- Kubernetes version (use `kubectl version`): 1.17.1
- OS (macOS 10.15, Windows 10, Ubuntu 19.10 etc): Windows 10

**Comment by wwitzel3** [comment]
This is probably related to #726

**Comment by abelal83** [comment]
@wwitzel3 thanks for quick response! I had a look at #726 but as it seemed related virtual nodes I opened this. 

Any eta when we can use fixed version?

On a personal note I'd like to say this is a brilliant tool (have just come across it), my developers aren't keen on using kubectl and octant is a perfect replacement for them! Thank you guys for this!

**Comment by wwitzel3** [comment]
Glad to hear you all are liking Octant!

The issue i referenced is for any container logs that don't have a leading timestamp, it was reported first on virtual nodes.

Can you provide some of the output from running the `kubectl logs` command? I'll be able to tell but looking at the output if this is related.

**Comment by abelal83** [comment]
of course, here's an output from one of our services. 
```
PS C:\Users\abx> kubectl logs serene-web-5b76858fcf-cb8rd -n serene-dev
[40m[32minfo[39m[22m[49m: Microsoft.AspNetCore.Mvc.Infrastructure.ObjectResultExecutor[1]
      Executing ObjectResult, writing value of type 'System.String'.
[40m[32minfo[39m[22m[49m: Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker[2]
      Executed action Serene.Health.HealthzController.Get (Serene.Web) in 0.2318ms
[40m[32minfo[39m[22m[49m: Microsoft.AspNetCore.Hosting.Internal.WebHost[2]
      Request finished in 0.8912ms 200 text/plain; charset=utf-8
[40m[32minfo[39m[22m[49m: Microsoft.AspNetCore.Hosting.Internal.WebHost[1]
      Request starting HTTP/1.1 GET http://serene-dev.sea.live/healthz
[40m[32minfo[39m[22m[49m: Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker[3]
      Route matched with {action = "Get", controller = "Healthz"}. Executing controller action with signature System.String Get() on controller Serene.Health.HealthzController (Serene.Web).
[40m[32minfo[39m[22m[49m: Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker[1]
      Executing action method Serene.Health.HealthzController.Get (Serene.Web) - Validation state: Valid
[40m[32minfo[39m[22m[49m: Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker[2]
      Executed action method Serene.Health.HealthzController.Get (Serene.Web), returned result Microsoft.AspNetCore.Mvc.ObjectResult in 0.0073ms.
[40m[32minfo[39m[22m[49m: Microsoft.AspNetCore.Mvc.Infrastructure.ObjectResultExecutor[1]
      Executing ObjectResult, writing value of type 'System.String'.
```
happy to help with anything else.

**Comment by wwitzel3** [comment]
Based on that output, i think #726 will fix your issue as well.

**Comment by wwitzel3** [comment]
I merged the fix so you will be able to try it out from the latest nightly build or building master from source yourself.

**Comment by shinebayar-g** [comment]
I see no logs in octant v0.12.1

`kubectl logs pod-name` returns history of logs, but doesn't follow as expected.
`kubectl logs -f pod-name` doesn't return anything.

**Comment by wwitzel3** [comment]
@shinebayar-g We have released Octant 0.13.1, I'd recommend trying again with the latest version. If you are still having issues, let us know.

If you can provide the console output for Octant when you are using the log viewer, that would be helpful.

Also, clear your browsers cache and retry viewing the logs. We made some updates to the log viewer that may be causing them not to show due to a component mismatch.
